app/transactions/transactions.html

```diff
@@ -6,6 +6,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="../../assets/stylesheets/shared/constants.css">
     <link rel="stylesheet" href="../../assets/stylesheets/shared/navbar.css">
+    <link rel="stylesheet" type="text/css" href="../../assets/stylesheets/transactions/transactions.css">
```

Comment:
either remove `type="text/css" ` or add it to each and every link for each of our pages to be consistent